### PR TITLE
[cilium-hubble] Add name to VM pod and change its icon

### PR DESCRIPTION
**Pull request by tarabrind**
## Description
Improved VM pod appearance in Hubble UI. Now it isn't an "Unknown App", but some VM with name and proper icon.

Hubble UI will:
- Use the label value `vm.kubevirt.internal.virtualization.deckhouse.io/name=<name>` as the name.
- Use the presence of the label `kubevirt.internal.virtualization.deckhouse.io=virt-launcher` to change the icon.

## Why do we need it, and what problem does it solve?
In the Hubble UI, the virtual machine have "Unknown App" instead of a name and have the standard Kubernetes pod icon.
image

## What is the expected result?
In the Hubble UI, the virtual machine will have the correct name and icon.
image

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cilium-hubble
type: chore
summary: Improved VM pod appearance in Hubble UI. Now it isn't an "Unknown App", but some VM with name and proper icon.
impact_level: default
```

### CODE REVIEW

File: modules/500-cilium-hubble/images/ui-frontend/patches/hubble-ui-vm-name.patch [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+   PrometheusApp = 'k8s:app=prometheus',
++  D8VirtualMachine = 'k8s:kubevirt.internal.virtualization.deckhouse.io=virt-launcher',
+ }
+ 
+ export class Labels {
+@@ -35,6 +37,7 @@ export class Labels {
+     'k8s:',
+     'io.kubernetes.pod.',
+     'app.kubernetes.io/',
++    // 'vm.kubevirt.internal.virtualization.deckhouse.io/',
```
Comment (apolovov) [inline modules/500-cilium-hubble/images/ui-frontend/patches/hubble-ui-vm-name.patch:43]: ??

File: modules/500-cilium-hubble/images/ui-frontend/patches/hubble-ui-vm-name.patch [hunk 2]
```diff
@@ -0,0 +1,86 @@
+diff --git a/src/components/EndpointCardHeader/helpers.ts b/src/components/EndpointCardHeader/helpers.ts
```
Comment (apolovov) [inline modules/500-cilium-hubble/images/ui-frontend/patches/hubble-ui-vm-name.patch:1]: Also, need a README.md file with patch description.
Like [this one](https://github.com/deckhouse/deckhouse/blob/main/modules/021-cni-cilium/images/bin-cilium/patches/README.md).

**Review by apolovov — CHANGES_REQUESTED** [review]

File: modules/500-cilium-hubble/images/ui-frontend/patches/hubble-ui-vm-name.patch (hunk 2 above)
Comment (tarabrind) [reply modules/500-cilium-hubble/images/ui-frontend/patches/hubble-ui-vm-name.patch]: README.md is added.

**Review by apolovov — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/yandex-cloud'...

File: modules/500-cilium-hubble/images/ui-frontend/patches/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+## 001-hubble-ui-vm-name-icon.patch
+
+Improved VM pod appearance in hubble-ui. Now it isn't an "Unknown App", but some VM with name and proper icon.
+
+Hubble UI:
+
+- Uses the label value `vm.kubevirt.internal.virtualization.deckhouse.io/name=<name>` as the name.
+- Uses the presence of the `label kubevirt.internal.virtualization.deckhouse.io=virt-launcher` to change the icon.
+
+> **NOTE:**  There is a SVG-file in patch.
```
Comment (apolovov) [inline modules/500-cilium-hubble/images/ui-frontend/patches/README.md]:
```suggestion
> **NOTE:**  There is a SVG-file in the patch.
```

**Pull request merged**